### ISSUE TITLE
Tests raising error 

### ISSUE THREAD

**Issue by Zapiano** [open-AIMS/ADRIA.jl#590]
I run `(ADRIA) pkg> test` and got the following error:

```
Running 32 scenarios for RCP 45   6%|█████████                                                                                                                           |  ETA: 0:08:25

ERROR: LoadError: type LOPCOWResult has no field bestIndex
Stacktrace:
  [1] getproperty
    @ .\Base.jl:37 [inlined]
  [2] retrieve_ranks(S::Matrix{Float64}, site_ids::Vector{Float64}, weights::Vector{Float64}, mcda_func::Type{JMcDM.LOPCOW.LOPCOWMethod})
    @ ADRIA.decision C:\Users\pribeiro\AIMS\Code\ADRIA.jl\src\decision\dMCDA.jl:275
  [3] rank_sites!(S::Matrix{Float64}, weights::Vector{Float64}, rankings::Matrix{Int64}, n_site_int::Int64, mcda_func::Type{JMcDM.LOPCOW.LOPCOWMethod}, rank_col::Int64)
    @ ADRIA.decision C:\Users\pribeiro\AIMS\Code\ADRIA.jl\src\decision\dMCDA.jl:228
  [4] guided_site_selection(d_vars::ADRIA.decision.DMCDA_vars, alg_ind::Int64, log_seed::Bool, log_shade::Bool, pref_seed_locs::Vector{Int64}, pref_shade_locs::Vector{Int64}, rankings_in::Matrix{Int64}, in_conn::Vector{Float64}, out_conn::Vector{Float64}, strong_pred::Vector{Int64}; methods_mcda::Vector{Any})
    @ ADRIA.decision C:\Users\pribeiro\AIMS\Code\ADRIA.jl\src\decision\dMCDA.jl:668
  [5] guided_site_selection
    @ C:\Users\pribeiro\AIMS\Code\ADRIA.jl\src\decision\dMCDA.jl:547 [inlined]
  [6] run_model(domain::ADRIADomain{NamedDims.NamedDimsArray{(:Source, :Receiving), Float64, 2, AxisKeys.KeyedArray{Float64, 2, Matrix{Float64}, Tuple{Vector{String}, Vector{String}}}}, NamedDims.NamedDimsArray{(:species, :sites), Float32, 2, AxisKeys.KeyedArray{Float32, 2, Matrix{Float32}, Tuple{UnitRange{Int64}, Vector{String}}}}, DataFrame, Matrix{Float64}, NamedDims.NamedDimsArray{(:timesteps, :sites, :scenarios), Float32, 3, AxisKeys.KeyedArray{Float32, 3, Array{Float32, 3}, Tuple{UnitRange{Int64}, Vector{String}, UnitRange{Int64}}}}, NamedDims.NamedDimsArray{(:timesteps, :sites, :scenarios), Float32, 3, AxisKeys.KeyedArray{Float32, 3, Array{Float32, 3}, Tuple{UnitRange{Int64}, Vector{String}, UnitRange{Int64}}}}}, param_set::NamedDims.NamedDimsArray{(:factors,), Float64, 1, AxisKeys.KeyedArray{Float64, 1, SubArray{Float64, 1, Matrix{Float64}, Tuple{Int64, Base.Slice{Base.OneTo{Int64}}}, true}, Base.RefValue{Vector{String}}}}, corals::DataFrame, cache::NamedTuple{(:fec_all, :fec_scope, :recruitment, :dhw_step, :cov_tmp, :depth_coeff, :site_area, :site_k_area, :wave_damage, :dhw_tol_mean_log), Tuple{Matrix{Float64}, Matrix{Float64}, Matrix{Float64}, Vector{Float64}, Matrix{Float64}, Vector{Float64}, Matrix{Float64}, Matrix{Float64}, Array{Float64, 3}, Array{Float64, 3}}})
    @ ADRIA C:\Users\pribeiro\AIMS\Code\ADRIA.jl\src\scenario.jl:659

...
```

**Comment by Zapiano** [comment]
@ConnectedSystems  and @Rosejoycrocker  FYI

**Comment by ConnectedSystems** [comment]
FYI @Zapiano this is caused by the inclusion of a new method in JMcDM.jl and is addressed in #589
